lfr/7pin_cardinal_demo/tests

```diff
@@ -0,0 +1,54 @@
+[Tests]
+  [generate_mesh]
+    type = RunApp
+    executable_pattern = 'cardinal*'
+    input = 'NTmesh.i  HCmesh.i'
+    cli_args = '--mesh-only'
+  []
+  [syntax_griffin]
+    type = RunApp
+    input = 'NT.i'
+    check_input = true
+    executable_pattern = 'cardinal*'
```

Comment:
would this run with griffin? (turning off any potential multiapp)

Reply:
Probably not, I have not done any standalone runs. A total of 144 processes were used to run this Cardinal case, and of course nekRS is the most computationally demanding component. Only the syntax checks are initiated here.

Reply:
but you called it the 'griffin' test? is there any non-griffin object?